### ISSUE TITLE
filterText not showing up for CalciteCombobox when used in React

### ISSUE THREAD

**Issue by nikki-lea** [Esri/calcite-design-system#9752]
### Check existing issues

- [x] I have [checked for existing issues](https://github.com/Esri/calcite-design-system/issues) to avoid duplicates

### Actual Behavior

event.filterText is undefined when a onCalciteComboboxFilterChange is fired in an CalciteCombobox component in react.

### Expected Behavior

event.filterText should be the input text

### Reproduction Sample

https://codesandbox.io/p/devbox/simple-combobox-df6kft?file=%2Fsrc%2FApp.jsx%3A21%2C9-21%2C38

### Reproduction Steps

1. Enter any text in the box
2. Note that the value for event.target.filterText is undefined

### Reproduction Version

2.10.0

### Priority impact

impact - p3 - not time sensitive

### Calcite package

- [ ] @esri/calcite-components
- [ ] @esri/calcite-components-angular
- [x] @esri/calcite-components-react
- [ ] @esri/calcite-design-tokens
- [ ] @esri/eslint-plugin-calcite-components

### Esri team

ArcGIS Enterprise

**Comment by nikki-lea** [comment]
> [@nikki-lea](https://github.com/nikki-lea) I haven't been able to reproduce this. Could you [test with the latest version](https://codesandbox.io/p/devbox/simple-combobox-forked-x46gpl?file=%2Fsrc%2FApp.jsx%3A9%2C31) and confirm if this is still an issue?

Nope, I can confirm that in the latest version this has been resolved so this is no longer an issue, thanks!

**Comment by jcfranco** [comment]
Awesome. Closing based on ☝️. Thanks for testing!
